DSMRloggerWS.ino

```diff
@@ -651,7 +655,8 @@ txtResponseMindergas[0] = '\0';
   oled_Print_Msg(1, "Verbinden met WiFi", 500);
 #endif  // has_oled_ssd1306
   digitalWrite(LED_BUILTIN, LED_ON);
-  startWiFi();
+  if (WiFi.status() != WL_CONNECTED)  //if NOT connected start WiFi
```

Comment:
Waarom doe je dit????

Reply:
Als telnet de WiFi start en het lukt. Dan is het starten van WiFi overbodig. Tevens disconnect StartWifi de verbinding terwijl een werkende verbinding is. Wellicht moet ik StartWifi gewoon aanpassen.